libcylinder/src/error.rs

```diff
@@ -89,12 +89,43 @@ impl std::fmt::Display for SignatureParseError {
 
 /// An error that can occur when loading a key
 #[derive(Debug)]
-pub struct KeyLoadError(pub String);
+pub struct KeyLoadError {
```

Comment:
This should probably also be guarded by the `"key-load" feature.

Perhaps this is for a future PR, but should this also be in an `key::error` module?

Reply:
Fixed

Reply:
I agree `KeyLoadError` should be moved to its own module, I can do that in a separate PR